### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by dktapps**
Enjoy!

### CODE REVIEW

File: src/pocketmine/lang/locale/cs-CZ.ini [hunk 1]
```diff
@@ -203,7 +203,6 @@ pocketmine.server.networkError=[Network] Zastavil interface {%0} protože {%1}
 pocketmine.server.networkStart=Startuji server na {%0}:{%1}
 pocketmine.server.info=Tento server běží na {%0} verzi {%1}"{%2}" (API {%3})
 pocketmine.server.info.extended=Tento server používá {%0} {%1}「{%2}」implantována API {%3} pro Minecraft:PE {%4}
-(protokol {%5})
```
Comment (dktapps) [inline src/pocketmine/lang/locale/cs-CZ.ini]: WTFFFFFFFFFF

File: src/pocketmine/lang/locale/ko-KR.ini [hunk 2]
```diff
@@ -188,13 +188,16 @@ pocketmine.data.playerCorrupted=플레이어 "{%0}"님의 데이터가 손상되
 pocketmine.data.playerOld=플레이어 "{%0}"님의 데이터가 오래되었습니다. 프로파일을 업데이트합니다.
 pocketmine.data.saveError=플레이어 "{%0}"님의 데이터를 저장하지 못했습니다: {%1}
 
-pocketmine.level.notFound="{%0}" 월드를 찾지 못했습니다
+pocketmine.level.notFound={%0}" 월드를 찾지 못했습니다
+pocketmine.level.loadError=월드
```
Comment (dktapps) [inline src/pocketmine/lang/locale/ko-KR.ini]: /suicide

File: src/pocketmine/lang/locale/pt-PT.ini [hunk 3]
```diff
@@ -287,7 +287,7 @@ pocketmine.command.whitelist.description=Gerencia a lista de jogadores permitido
 
 pocketmine.crash.create=Ocorreu um erro irrecuperável e o servidor teve despejo de memória (crash dump). Criando um relatório do despejo de memória
 pocketmine.crash.error=Não foi possível criar o relatório de despejo de memória (crash dump): {%0}
-pocketmine.crash.submit=Por favor, faça upload do arquivo "{%0}" na página de reportar bugs e problemas http://crash.pocketmine.net/ e dê o máximo de informações possível.
+pocketmine.crash.submit=Por favor, faça upload do arquivo "{%0}" na página de reportar bugs e problemas http:
```
Comment (dktapps) [inline src/pocketmine/lang/locale/pt-PT.ini]: -_-

File: src/pocketmine/lang/locale/he-IL.ini [hunk 4]
```diff
@@ -252,7 +252,6 @@ usage_disable=אתה רוצה לכבות את השימוש האנונימי?
 ip_get=משיג את כתובת הIP החיצונית והפנימית שלך
 ip_warning=הIP החיצוני שלך הוא  {{EXTERNAL_IP}} אולי תצטרך לעשות פורט פורווארדינג לIP הפנימי שלך {{INTERNAL_IP}}
 ip_confirm=תיהיה בטוח לבדוק אם אתה צריך לעשות פורט פורווארדינג ואתה לא עושה שום שחקן חיצוני לא יוכל להצטרף
-[לחץ אנטר]
```
Comment (dktapps) [inline src/pocketmine/lang/locale/he-IL.ini]: -_-

File: src/pocketmine/lang/locale/id-ID.ini [hunk 5]
```diff
@@ -202,7 +202,7 @@ pocketmine.server.start=Memulai Minecraft:PE server versi {%0}
 pocketmine.server.networkError=[Network] Memberhentikan antar muka {%0} disebabkan oleh {%1}
 pocketmine.server.networkStart=Membuka server dalam {%0}:{%1}
 pocketmine.server.info=Server ini berjalan dengan {%0} Versi {%1} "{%2}" (API {%3})
-pocketmine.server.info.extended=Server ini berjalan dengan versi {%0} {%1} 「{%2} menggunakan versi API {%3} for Minecraft: PE {%4} (protocol version {%5}) bahasa by icsmoke
+pocketmine.server.info.extended=Server ini berjalan dengan versi {%0} {%1} 「{%2} menggunakan versi API {%3} for Minecraft: PE {%4} (protocol version {%5})
```
Comment (dktapps) [inline src/pocketmine/lang/locale/id-ID.ini]: ?????

**Comment by robske110** [comment]
Travis is having fun

**Comment by dktapps** [comment]
Stupid Crowdin... might just get rid of it. All this mess and I only changed 3 lines.

File: src/pocketmine/lang/locale/en-US.ini [hunk 6]
```diff
@@ -1,361 +1,45 @@
...
-
-commands.particle.success=Playing effect {%0} for {%1} times
-commands.particle.notFound=Unknown effect name {%0}
-
-commands.players.usage=/list
-commands.players.list=There are {%0}/{%1} players online:
-
-commands.kill.successful=Killed {%0}
-
 commands.banlist.ips=There are %d total banned IP addresses:
```
Comment (dktapps) [inline src/pocketmine/lang/locale/en-US.ini]: I swear to God...

**Comment by MrPowerGamerBR** [comment]
This is one of those moments that I regret watching the PocketMine-MP repo... now my GitHub home page isn't a home page, it is the "New translations" page. (with sometimes a random comment by @dktapps about the translations)

**Comment by dktapps** [comment]
yeah, sorry about that... you should've seen the spam in Slack

**Pull request closed without merging**